gems/canvas_sanitize/lib/canvas_sanitize/canvas_sanitize.rb

```diff
@@ -90,6 +90,7 @@ def self.clean_style_value(config, value)
                    'role',
                    'lang',
                    'dir',
+                   :data,
```

Comment:
Thanks for taking a look, @claydiffrient. The sanitize gem uses `:data` as a symbol to differentiate it from an attribute named 'data'.

> To allow arbitrary HTML5 data-* attributes, use the symbol :data in place of an attribute name.